html/about.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>The Kingdom Comes Global Worship Conference 2023</title>
+  <script
+  src="https://kit.fontawesome.com/6eed52bc92.js"
+  crossorigin="anonymous"
+></script>
+<link rel="stylesheet" href="/css/main.css" />
+</head>
+<body>
+  <!-- ==== Primary and secondary navigation menu start here ==== -->
+  <header class="main-header">
+    <div class="secondary-navbar">
+      <ul class="secondary-navbar-content flex">
+        <li>
+          <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li><a href="#"></a>Arabic</li>
+        <li><a href="#"></a>My Page</li>
+        <li><a href="#"></a>Logout</li>
+      </ul>
+    </div>
+    <nav class="primary-navbar flex">
+      <a href="index.html"
+        ><img
+          class="logo"
+          src="/images/The-Kingdom-T-Logo.png"
+          alt="The Kingdom Logo"
+      /></a>
+      <ul class="nav-menu flex">
+        <li class="nav-item">
+          <a class="nav-link active" href="#">About</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="index.html#program">Program</a>
+        </li>
+        <li class="nav-item"><a class="nav-link" href="#">Join</a></li>
+        <li class="nav-item">
+          <a class="nav-link" href="index.html#partners">Sponsor</a>
+        </li>
+        <li class="nav-item"><a class="nav-link" href="#">News</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Compaign</a></li>
+      </ul>
+      <div class="hamburger">
+        <span class="line"></span>
+        <span class="line"></span>
+        <span class="line"></span>
+      </div>
+    </nav>
+  </header>
+      <!-- ==== Primary and secondary navigation menu ends here ==== -->
+      <main id="home">
+        <section class="about headline flex">
+          <div class="about headline-content flex">
+            <h1>The Kingdom Comes Global Worship Conference 2023</h1>
```

Comment:
- [ ] I think make the font weight for the heading text greater to be the same as the original design 👌🏻

Reply:
Thanks!